tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs

```diff
@@ -18,7 +18,7 @@ namespace Lykke.Icon.Sdk.Tests
 {
     [SuppressMessage("ReSharper", "StringLiteralTypo")]
     [SuppressMessage("ReSharper", "CommentTypo")]
-    public class IconServiceVCRTest : TestWithEnvVariables
+    public class IconServiceVCRTest //: TestWithEnvVariables
```

Comment:
Commented code here and below

Reply:
One day we will revert this(maybe for local testing)

Reply:
Not sure about it, but ok :)